src/bdk.udl

```diff
@@ -168,6 +168,11 @@ dictionary LocalUtxo {
   boolean is_spent;
 };
 
+dictionary AddressAmount {
+ string address;
+ u64    amount;
```

Comment:
I don't mind the idea of aligning the fields/methods after the type declaration, but because we don't do it anywhere else in the file I think this should be set similar to the other dictionaries:
```
dictionary AddressAmount {
  string address;
  u64 amount;
}
```

And the realignment be left for a separate PR if it's really where we want to go (but the UDL files are going away anyway so I don't think it's worth considering too hard).

Also note that you need 2 spaces for correct alignment.

Reply:
Good catch, yes I agree the formatting/spacing for the `AddressAmount` dictionary should be formatted and line up with the rest of the UDL file. Unfortunately `cargo fmt` won't do it so it needs to be done manually.

Reply:
I don't remember doing this intentionally but you are right lets keep things uniform. Proposed changes have been updated.